nefertari/view.py

```diff
@@ -176,10 +176,15 @@ def get_collection_es(self):
         results for default response renderers to work properly.
         """
         from nefertari.elasticsearch import ES
-        return ES(self.Model.__name__).get_collection(**self._query_params)
+        params = self._query_params.copy()
+
+        if ES.settings.asbool('acl_filtering'):
+            params['_identifiers'] = self.request.effective_principals
```

Comment:
Because in Pyramid they are called "principal identifiers".
E.g. here http://docs.pylonsproject.org/projects/pyramid//en/latest/api/request.html#pyramid.request.Request.effective_principals